.circleci/config.yml

```diff
@@ -5,20 +5,12 @@ orbs:
     go: circleci/go@1.1.1
 
 parameters:
-  registry:
```

Comment:
this "separation" of reistry, org, etc was needed for the now removed docker/publish orb step, now we can just have 1 parameter containing the full image name:tag